### PR TITLE
Quora search module added. 

### PR DESCRIPTION
**Pull request by Pk13055**
I have added Quora functionality. However, the main answer content isn't parsing properly. I will add another commit to fix that soon. In the meanwhile, if @nikhilrayaprolu  could review the existing work and suggest any changes.

### CODE REVIEW

**Comment by Pk13055** [comment]
@nikhilrayaprolu Could you please review?

File: app.json [hunk 1]
```diff
@@ -1,29 +0,0 @@
-{
```
Comment (AnshulMalik) [inline app.json]: Why delete this file?

File: app/scrapers/yahoo.py [hunk 2]
```diff
@@ -34,10 +34,10 @@ def results_search(self,query):
                 r = y.get('href')
```
Comment (AnshulMalik) [inline app/scrapers/yahoo.py]: This is a PR for quora. This change should come in another PR.

File: app/scraper.py [hunk 3]
```diff
@@ -1,171 +0,0 @@
-from __future__ import print_function
```
Comment (AnshulMalik) [inline app/scraper.py]: Looks like you created one file for each scraper but you have not added other scrapers here.

File: app.json (hunk 1 above)
Comment (Pk13055) [reply app.json]: I deleted this by mistake. I shall add it again.

File: app/scraper.py (hunk 3 above)
Comment (Pk13055) [reply app/scraper.py]: This has already been merged. This file has been removed, and the initialization has been added to `__init__.py` (check  #141)

File: app/scrapers/yahoo.py (hunk 2 above)
Comment (Pk13055) [reply app/scrapers/yahoo.py]: This is not a change. I was trying to fix merge conflicts, that's why I made this change.

**Comment by Pk13055** [comment]
Given the conflicts and the increasing rebases, I am closing this PR and opening a new one.

**Pull request closed without merging**